packages/lesswrong/components/posts/PostsItem2.jsx

```diff
@@ -177,6 +191,45 @@ const styles = (theme) => ({
       display: "block"
     }
   },
+  nextUnreadIn: {
+    color: theme.palette.grey[800],
+    
+    [theme.breakpoints.up('md')]: {
+      position: "absolute",
+      left: 42,
+      top: 30,
+      zIndex: 1000,
+    },
+    [theme.breakpoints.down('sm')]: {
+      order: -1,
+      width: "100%",
+      marginTop: -10,
+    },
+    
+    "& a": {
+      color: theme.palette.primary.main,
+    },
+  },
+  sequenceImage: {
+    marginTop: -12,
```

Comment:
Also feels very fragile here. I am confused why this has a `top: 4` but a `marginTop: -12`. Shouldn't those partially cancel out?

Reply:
The `marginTop` cancels out the top-padding of `.PostsItem2-postsItem`. Top `top: 4` is empirically necessary, but I don't have a principled reason why. But... it looks like specifying the height on both the image and its container (rather than just the image, makes it line up correctly again; there were 4px of height mismatch that were neither margin, padding, nor border separating the image's height from the container's height.